tests/integrations/gcp/test_gcp.py

```diff
@@ -81,6 +81,9 @@ def init_sdk(timeout_warning=False, **extra_init_args):
         transport=TestTransport,
         integrations=[GcpIntegration(timeout_warning=timeout_warning)],
         shutdown_timeout=10,
+        # excepthook -> dedupe -> event_processor client report gets added
+        # which we don't really care about for these tests
+        send_client_reports=False,
```

Comment:
I just turned client reports off for the GCP tests since the actual problem is unrelated and caused by the same exception being caught twice and dropped by the `DedupeIntegration` event processor.

I added a separate test for the case below.